src/main/scala/is/hail/table/Table.scala

```diff
@@ -912,8 +915,7 @@ class Table(val hc: HailContext,
     new MatrixTable(hc,
       matrixType,
       MatrixLocalValue(Annotation.empty,
-        colIDs,
-        Annotation.emptyIndexedSeq(ncols)),
+        colIDs.map(Annotation(_))),
```

Comment:
Can we make the colKeyExpr => colKeyExprs so it is compound out of the gate?

Reply:
can I do this later?